mrtt-ui/src/components/RestorationAimsForm/restorationAimsQuestionsAndAnswers.js

```diff
@@ -0,0 +1,67 @@
+export default {
+  1: {
+    question: '3 .1 What are the ecological aim(s) of the project activities at the site?',
+    answers: [
+      'Increase mangrove area',
+      'Improve mangrove condition/halt or reduce degradation',
+      'Increase mangrove species richness',
+      'Offset mangrove loss from another area',
+      'Habitat protection',
+      'Increase native fauna/wildlife',
+      'Increase native flora/vegetation (non-mangrove)',
+      'Reduce invasive species',
+      'Increase ecological resilience',
+      'Increase habitat connectivity',
+      'Restore hydrological connectivity',
+      'Improve sediment dynamics',
+      'Improve nutrient cycling',
+      'Increase carbon storage and sequestration',
+      'None',
+      'Unknown',
+      'Other'
+    ]
```

Comment:
ideally we keep all these in one spot, are you ok with adding this in the same structure as `questionMapping.js`?